### PR TITLE
Added a method to convert a position (number of totes we wish to stack u...

### PR DESCRIPTION
**Pull request by BenDeVries**
...pon) to the height of the position.

### CODE REVIEW

File: RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java [hunk 1]
```diff
@@ -100,18 +100,15 @@ public void changeControlModeWinch(ControlMode mode) {
      * Stops the elevator from moving. Used at the end of commands.
      */
     public void stopElevator() {
-    	// stops any current commands telling the elevator to move.
     	winch.disableControl();
     	System.out.println("Elevator has stopped.");
     }
     
     /**
      * TODO Make this actually work - It will drift and continually use a new point to hold the position
```
Comment (phroa) [inline RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java:110]: did you mean to remove this line or keep it?

File: RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java [hunk 2]
```diff
@@ -146,8 +142,30 @@ public void setPosition(double position) {
...
+			break;
+		case 4:
+			positionNumber = POSITION_FOUR;
+			break;
+		case 5:
+			positionNumber = POSITION_FIVE;
+			break;
+		case 6:
+			positionNumber = POSITION_SIX;
+			break;
```
Comment (phroa) [inline RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java:169]: I'd add a `default:` case to this - `positionNumber` isn't guaranteed to be `0 <= x <= 6`

File: RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java (hunk 1 above)
Comment (BenDeVries) [inline RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java:110]: I did want to delete this comment, a description of the method is present in the javadoc above.

Comment (phroa) [inline RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java:110]: I was referring to the `TODO` on line `109` (`108`?)

Comment (BenDeVries) [inline RobotCode/MecanumDrive/src/org/usfirst/frc4915/MecanumDrive/subsystems/Elevator.java:110]: Oh, sorry, keep that todo, the current code will still not work.

**Pull request merged**